code/modules/projectiles/projectile/bullets.dm

```diff
@@ -115,16 +115,17 @@
 	weaken = 0
 
 /obj/item/projectile/bullet/fourtyfive //secgun ammo!
-	damage = 15
-	stun = 2 //stopping power
-	weaken = 2
+	damage = 13
+	drowsy = 0.25 //stopping power
```

Comment:
Agony seemed to cause pain / slumping in testing. Drowsy's there to slow down the target a bit. It's one tick of drowsy, which is enough. I could just up it to one.